src/Destination/Adapter/RestApiOutput.php

```diff
@@ -85,19 +86,19 @@ private function call($config, $values = [])
                 $serviceUrl
             ));
 
-            $this->output->writeln(' -- Body: ' . $body);
+            $this->output->writeln(' -- Body: '.$body);
```

Comment:
Call to method `writeln` from undeclared class `\Symfony\Component\Console\Output\OutputInterface`